pkgs/misc/emulators/dosemu2/default.nix

```
@@ -0,0 +1,73 @@
+{ stdenv, fetchFromGitHub, fetchurl,
+bison, flex, SDL2, udev, libslirp, json_c, autoconf, automake,
+pkg-config, libbsd, hexdump, slang, xorg, readline, libao,
+gpm, munt, libieee1284 }:
+stdenv.mkDerivation {
+  name = "dosemu2";
```

```suggestion
  pname = "dosemu2";
  version = "X.X.X"
```

Please specify a version.

---

The source is a recent git commit, not a released version.

---

```suggestion
  pname = "dosemu2";
  version = "unstable-XXXX-XX-XX";
```
Then please set version to the date of the commit.